eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -161,10 +167,10 @@ jobs:
   - script: |
       set -x
 
-      customPrepArgs=""
+      customPrepArgs="--artifacts-rid ${{ parameters.artifactsRid }}"
```

Comment:
Any reason why we don't utilize the default instead of specifying it for most of the CI legs.  This would eliminate duplication but more importantly we would be invoking the script in the canonical way that we expect most customers to use the script.

Reply:
> Any reason why we don't utilize the default instead of specifying it for most of the CI legs.

I was following the pattern used by the architecture parameter; architecture does not specify a default but is `x64` for all legs except for `Ubuntu2204Arm64_Offline_MsftSdk`. artifactsRid is similar in that it is `centos.8-x64` for all legs except `Alpine317_Offline_PreviousSourceBuiltSdk`.

That said, I agree that it makes more sense to utilize the default for artifactsRid, especially since it would allow us to invoke the script in the canonical way as you pointed out.

Reply:
The difference here is that the prep script defines the default which will change over time - e.g. centos.9.  Alpine is the exception.  I would like to avoid the maintenance and risk of missing something when the default is updated.

Reply:
Refactored for default artifactsRidin https://github.com/dotnet/installer/pull/17370/commits/bf697aa5cab242f80871e0db905b3d85ba24dedf. The issue is that there's still multiple places where the default is defined/checked, but there's less places where this occurs than before.